modules/lighthouse_service_biosero.js

```diff
@@ -101,35 +101,55 @@ const createDestinationPlateBiosero = async (form) => {
  * @returns
  */
 const failDestinationPlateBiosero = async (form) => {
-  try {
-    const url = `${config.privateRuntimeConfig.lighthouseBaseURL}/events`
-    const body = {
-      event_type: 'lh_biosero_cp_destination_plate_failed',
-      barcode: form.barcode,
-      user_id: form.username,
-      failure_type: form.failureType,
-    }
+  const url = `${config.privateRuntimeConfig.lighthouseBaseURL}/events`
+  const body = {
+    event_type: 'lh_biosero_cp_destination_plate_failed',
+    barcode: form.barcode,
+    user_id: form.username,
+    failure_type: form.failureType,
+  }
 
-    const headers = { headers: { Authorization: config.privateRuntimeConfig.lighthouseApiKey } }
+  const headers = { headers: { Authorization: config.privateRuntimeConfig.lighthouseApiKey } }
 
+  try {
     const response = await axios.post(url, body, headers)
-
-    if (response._status === 'OK') {
-      // success
+    if (response.data._status === 'OK') {
+      // successfull insert of fail event
       return {
         success: true,
         response: `Successfully failed destination plate with barcode: ${form.barcode}`,
       }
     } else {
-      const errors = response._error
-      // failure
+      let error
```

Comment:
Think this can be put into a single line something like:
`let error = response.data?._error.message || response._error`